examples/model_selection/plot_cost_sensitive_learning.py

```diff
@@ -664,10 +664,11 @@ def business_metric(y_true, y_pred, amount):
 #
 # Here, we will reuse the decision threshold found in the previous section to create a
 # new model and check that it gives the same results.
+from sklearn.frozen import FrozenEstimator
```

Comment:
I think that we need to add a small explanation above to explain that we need also to freeze the estimator to avoid refitting it. We should have explain the option `prefit=True` but I forgot it in the past iteration.

It only needs to be a single sentence.